.travis.yml

```diff
@@ -14,12 +14,7 @@ php:
     - hhvm
 
 env:
-    global:
-        - TEST_COMMAND="composer test"
-    matrix:
-        - SYMFONY_VERSION=3.2.*
-        - SYMFONY_VERSION=3.1.*
-        - SYMFONY_VERSION=2.8.*
+    - SYMFONY_VERSION=2.8.* TEST_COMMAND="composer test"
```

Comment:
I would maybe also set `minimum-stability` in the `composer.json` file to `dev` so that you would get a build with the latest Symfony development version for free ("normal" builds could use the `--prefer-stable` flag).